stackit/internal/services/rabbitmq/instance/resource.go

```diff
@@ -52,12 +52,36 @@ type Model struct {
 
 // Struct corresponding to DataSourceModel.Parameters
 type parametersModel struct {
-	SgwAcl types.String `tfsdk:"sgw_acl"`
+	SgwAcl               types.String `tfsdk:"sgw_acl"`
+	ConsumerTimeout      types.Int64  `tfsdk:"consumer_timeout"`
+	EnableMonitoring     types.Bool   `tfsdk:"enable_monitoring"`
+	Graphite             types.String `tfsdk:"graphite"`
+	MaxDiskThreshold     types.Int64  `tfsdk:"max_disk_threshold"`
+	MetricsFrequency     types.Int64  `tfsdk:"metrics_frequency"`
+	MetricsPrefix        types.String `tfsdk:"metrics_prefix"`
+	MonitoringInstanceId types.String `tfsdk:"monitoring_instance_id"`
+	Plugins              types.List   `tfsdk:"plugins"`
+	Roles                types.List   `tfsdk:"roles"`
+	Syslog               types.List   `tfsdk:"syslog"`
+	TlsCiphers           types.List   `tfsdk:"tls_ciphers"`
+	TlsProtocols         types.String `tfsdk:"tls_protocols"`
```

Comment:
It's a bit strange that this is plural, right? But it's the API field name

Reply:
Very weird 🤷